releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,33 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.0 - January 13, 2020
+
+* Simplified Feature Management config file format.
+
+```yaml
+feature-management:
+  Beta: true
+  DarkTheme:
+    EnabledFor:
```

Comment:
Don't we have the Java-style for this, for example, `enabled-for`?

Reply:
This is running into issues as it doesn't work well supporting both data coming from both App Config and a configuration file as we have to process them differently. When coming from a file "." is not a valid character to have inside of a feature name. i.e.

```yaml
feature-management:
 feature.a: true
```

This value coming from the file is read as:

```yaml
feature-management:
  feature:
    a: true
```

But this just works coming from App Config. The bigger issue where `enabled-for` comes in is that coming from one it is read as a list of items i.e: [{Name:"Random", Parameters:[...]}, {Name:"ClientFilter", Parameters:[...]}], but the other reads them as a HashMap to fake being a list. {0:{Name:"Random", Parameters:[...]}, 1:{Name:"ClientFilter", Parameters:[...]}}.

I am looking into beter ways to solve this list hashmap difference. Which results in us using enabledFor and EnabledFor to seperate the two and allow support for enabled-for as right now it doesn't seem to allow for it to be set.

Reply:
If we are not ready to announce this, let's don't mention it this time.